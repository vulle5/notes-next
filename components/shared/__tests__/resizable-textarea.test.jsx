import React from 'react'
import { render } from '@testing-library/react'

import ResizableTextarea from '../resizable-textarea'

test('renders correctly', () => {
  const { container } = render(<ResizableTextarea />)
  expect(container.firstChild).toMatchSnapshot()
})

test('has correct default props', () => {
  expect(ResizableTextarea.defaultProps).toBeDefined()
  expect(ResizableTextarea.defaultProps).toStrictEqual({
    minRows: 5,
    maxRows: 10,
    textareaRef: null
  })
})

describe('onChange', () => {
  // TODO: Implement tests
})
