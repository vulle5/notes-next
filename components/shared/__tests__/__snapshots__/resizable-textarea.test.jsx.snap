// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<textarea
  class="jsx-1612382694 "
  rows="5"
/>
`;
